src/routes/owners/owners.controller.ts

```diff
@@ -23,12 +22,39 @@ export class OwnersController {
     return this.ownersService.getSafesByOwner({ chainId, ownerAddress });
   }
 
-  @ApiOkResponse({ type: SafeList })
-  @Get('owners/:ownerAddress/safes')
-  async getAllSafesByOwner(
+  @ApiOkResponse({
+    schema: {
+      type: 'object',
+      additionalProperties: {
+        type: 'array',
+        items: { type: 'string' },
+      },
+    },
+  })
```

Comment:
The types were previously incorrect.

This cannot be set in a class as the dynamic properties are top-level.